mypyc/primitives/int_ops.py

```diff
@@ -131,13 +131,12 @@ def int_compare_op(op: str, c_func_name: str) -> None:
     emit=simple_emit('{dest} = {args[0]} + {args[1]};'))
 
 
-def int_unary_op(op: str, c_func_name: str) -> OpDescription:
```

Comment:
Some thoughts: I think the original design somehow reflects or predicts what we are doing now

Reply:
Yeah, the new approach fits in very nicely here.